prov/gni/test/freelist.c

```diff
@@ -149,7 +181,7 @@ Test(gnix_freelist, freelist_random_alloc_free)
 
 	ret = _gnix_fl_init(sizeof(struct list_ts),
 			     offsetof(struct list_ts, e),
-			     0, 0, 0, 0, &fl);
+			     0, 1, 0, 0, &fl);
```

Comment:
Since there is no default value anymore, we should test 1 and something bigger, like 23 (I like prime numbers).  I think this test an be changed to do both.